app/src/main/java/com/sdp/movemeet/view/activity/SplashActivity.java

```diff
@@ -0,0 +1,42 @@
+package com.sdp.movemeet.view.activity;
+
+import android.content.Intent;
+import android.nfc.cardemulation.HostNfcFService;
+import android.os.Bundle;
+import android.view.WindowManager;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.airbnb.lottie.LottieAnimationView;
+import com.sdp.movemeet.R;
+import com.sdp.movemeet.view.home.HomeScreenActivity;
+import com.sdp.movemeet.view.home.LoginActivity;
+
+public class SplashActivity extends AppCompatActivity {
+
+    private boolean isChecked = false;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_splash);
+
+        getWindow().addFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN);
+
+        Thread td = new Thread(){
+            public void run(){
+                try{
+                    sleep(4000);
+                }catch(Exception ex){
+                    ex.printStackTrace();
+                }
+                finally {
+                    Intent intent = new Intent(SplashActivity.this, HomeScreenActivity.class);
```

Comment:
Can't you directly call getApplicationContext(), instead of explicitely stating SplashActivity.this ? I know this is a formality, but this is how we've done it everywhere else.